### PR TITLE
Support building docs

### PR DESCRIPTION
**Pull request by icemac**
This sits on top of #43. The commits on this branch start with 58dbe25.
See the following examples built with this branch:
* https://github.com/zopefoundation/zope.browser/pull/13
* https://github.com/zopefoundation/zope.dublincore/pull/13 (includes (doc-) testing the documentation)

### CODE REVIEW

File: config/README.rst [hunk 1]
```diff
@@ -152,6 +162,12 @@ fail-under
 with-pypy
   Does the package support PyPy: True/False
 
+with-docs
+  Build the documentation via sphinx: True/False
+
+with-sphinx-doctests
+  Run the documentation as doctest using sphinx: True/False
```
Comment (mgedmin) [inline config/README.rst:169]: Capitalize Sphinx?

**Review by mgedmin — APPROVED** [review]
LGTM

and this is a great demonstration on why Jinja was a good idea, imagine doing this with %-formatting ;)

File: config/README.rst (hunk 1 above)
Comment (icemac) [reply config/README.rst]: Of course.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😀

**Pull request merged**